R/sun-methods.R

```diff
@@ -110,6 +161,10 @@ setMethod("sunriset", signature(crds="sf", dateTime="POSIXct"),
               res
           })
 
+#' @description Method for matrix objects. 
+#' @param crs A "CRS" object representing the coordinate reference system.
```

Comment:
CRS is old sp, it should be a `crs` object. I would insert a link here to the relevant function in sf using [crs][sf::st_crs], as in `A [crs][sf::st_crs] object representing the ....`